static/app/views/settings/organizationIntegrations/detailedView/useIntegrationFeatureProps.tsx

```diff
@@ -0,0 +1,29 @@
+import styled from '@emotion/styled';
+
+import type {IntegrationFeature} from 'sentry/types/integrations';
+import {singleLineRenderer} from 'sentry/utils/marked';
+import useOrganization from 'sentry/utils/useOrganization';
+
+export function useIntegrationFeatureProps({
+  featureData,
+}: {
+  featureData: IntegrationFeature[];
+}) {
+  const organization = useOrganization();
```

Comment:
doesn't seem like we use this for anything

Reply:
Sounds good, it doesn't get used but everywhere was spreading `...featureProps` expecting organization, but being explicit does sound better :+1: